creator-node/scripts/dbManagerClone.js

```diff
@@ -0,0 +1,52 @@
+class DBManager {
```

Comment:
don't intend to keep this file. basically copied this and added models as the last argument in createNewDataRecord because if i required the original dbManager from a script I got several convict config errors